perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -1501,7 +1501,7 @@ sub process_garden_modification : Private {
             $c->set_param('pro_rata', $pro_rata);
             $c->set_param('admin_fee', $cost_now_admin);
         }
-
+        $c->cobrand->call_hook(waste_garden_mod_params => $data);
```

Comment:
I'd put this below after setup_garden_sub_params, I think? Is matching what that is doing (when that applies to both new/modify) - and also needs to override the Container_Type/Quantity set by it (though is setting to the same values).